src/industry_gui.cpp

```diff
@@ -908,31 +921,11 @@ class IndustryViewWindow : public Window
 
 			CargoSuffix suffix;
 			GetCargoSuffix(CARGOSUFFIX_IN, CST_VIEW, i, i->type, ind, a.cargo, &a - i->accepted.data(), suffix);
+			/* if the industry is not stockpiling then don't amount on the acceptance display. */
```

Comment:
```suggestion
			/* if the industry is not stockpiling then don't show amount on the acceptance display. */
```
Not sure amount is the right verb.